kubeflow/common/iap-ingress/base/config-map.yaml

```diff
@@ -178,11 +178,11 @@ data:
         gcloud --project=${PROJECT} compute health-checks update http ${HEALTH_CHECK_URI} --request-path=${HEALTHCHECK_PATH}
```

Comment:
Should we change this to patch `iap-backendconfig` with ${HEALTHCHECK_PATH}?

Reply:
I am not sure. I haven't changed that flow cause I didn't know exactly its purpose.
If it is just a customisation point, shouldn't we just apply a kustomize overlay/patch (to change `BackendConfig`) ?

Reply:
Agree, looks like its obsolete now, I will remove.